### ISSUE TITLE
link to github

### ISSUE THREAD

**Issue by Martinligabue** [KnorpelSenf/cloudconvert-bot#21]
since the bot is now off maybe a link in the bot, like with a new command (under https://github.com/KnorpelSenf/cloudconvert-bot/blob/6f79039fbc51aa2b95e9d0f8e9a0ef13e47d37f2/src/bot/bot.ts#L138 I think), or in the contribute command might help people reach this repo

**Comment by KnorpelSenf** [comment]
Why would people need to reach the repo? You mean in order to make them contribute a rewrite?

**Comment by Martinligabue** [comment]
yeah, like I wanted to help update to apiv2, but didn't find any link from the bot to help
